docs/release-notes/1306-remove-tensorboard-source-column.txt

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**BUG FIXES**
+
+- Remove unused TensorBoard sources column from task list page.
```

Comment:
question: was this present in our last release? if not we don't need notes

Reply:
I'll omit it when I compile the notes, no need to restart tests

Reply:
🙏